ci/conda/meta.yaml

```diff
@@ -19,6 +19,7 @@ requirements:
     - ninja
     - occt ==7.3.0
     - tbb-devel
+    - pybind11
```

Comment:
i don't think this is needed as cloning pybind11 is in the appveyor/travis script...so far i'm cloning master so it can be used instead of whatever release is available on conda

Reply:
I'm getting the following error without this line:
```
-- Searching for pybind11...
CMake Error at CMakeLists.txt:210 (find_package):
  By not providing "Findpybind11.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "pybind11",
  but CMake did not find one.

  Could not find a package configuration file provided by "pybind11" with any
  of the following names:

    pybind11Config.cmake
    pybind11-config.cmake

  Add the installation prefix of "pybind11" to CMAKE_PREFIX_PATH or set
  "pybind11_DIR" to a directory containing one of the above files.  If
  "pybind11" provides a separate development package or SDK, be sure it has
  been installed.

```